spacy/tests/pipeline/test_spancat.py

```diff
@@ -188,7 +188,9 @@ def test_ngram_suggester(en_tokenizer):
 def test_ngram_sizes(en_tokenizer):
     # test that the range suggester works well
     size_suggester = registry.misc.get("spacy.ngram_suggester.v1")(sizes=[1, 2, 3])
-    range_suggester = registry.misc.get("spacy.ngram_range_suggester.v1")(min_size=1, max_size=3)
+    range_suggester = registry.misc.get("spacy.ngram_range_suggester.v1")(
+        min_size=1, max_size=3
+    )
```

Comment:
This is a good example where the code formatting highlights something that could probably be formatted better. So maybe we can make this:

```suggestion
    suggester_factory = registry.misc.get("spacy.ngram_range_suggester.v1")
    range_suggester = suggester_factory(min_size=1, max_size=3)
```

Let's see if we can actually push to this PR, that'd be super useful!